### ISSUE TITLE
Selecting a clip via tabbing and pressing Delete causes the entire track to be deleted

### ISSUE THREAD

**Issue by dozzzzer** [audacity/audacity#9741]
1. Create a track with a clip.
2. Press Tab to navigate to the clip and make it the focus.
3. Press Delete/Backspace.
4. Note: rather than deleting the clip, the track was deleted.

https://github.com/user-attachments/assets/182cd288-573d-4b31-8b14-7fdf4050be5d

**Comment by chinakov** [comment]
This is reproducible on the latest Master (f90f955b9dd9347f983fe8ad0c30dcbccff24ede)

**Comment by dozzzzer** [comment]
Talk to @Tantacrul , we need to fix how the clips are selected

**Comment by DilsonsPickles** [comment]
This should be fixed by Elnur's track navigation work. See ticket [here](https://github.com/audacity/audacity/pull/10287)

**Comment by chinakov** [comment]
Fixed iva https://github.com/audacity/audacity/pull/10287
